jumpscale/packages/vdc_dashboard/chats/discourse.py

```diff
@@ -26,6 +26,18 @@ def _configure_admin_username_password(self):
             else:
                 valid_password = True
 
+    def _ask_smtp_settings(self):
+        form = self.new_form()
+        smtp_host = form.string_ask("SMTP Host", default="smtp.gmail.com", required=True)
+        smtp_port = form.string_ask("SMTP Port", default=587, required=True)
+        smtp_username = form.string_ask("Email (SMTP username)", required=True)
+        smtp_password = form.secret_ask("Email Password", required=True, isAlphanumeric=True)
+        form.ask()
+        self.smtp_host = smtp_host.value
+        self.smtp_port = f'"{smtp_port.value}"'
```

Comment:
why?

Reply:
I think we did it like that to pass it to the helm command with double quotes `"576"`

Reply:
> I think we did it like that to pass it to the helm command with double quotes `"576"`

I think if u use a `backslash` it will be more clear, what's ur opinion? @RafyAmgadBenjamin 
```
self.smtp_port = f'\"{smtp_port.value}\"'
```